<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResizeNode</class>
 <widget class="QWidget" name="ResizeNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>输出尺寸</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>宽度：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>高度：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="width_spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>480</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="height_spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>640</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="ratio_comboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>4 : 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16 : 9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 : 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 : 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 : 3</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ratio_checkBox">
          <property name="text">
           <string>比例</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>插值方式：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="interpolation_comboBox">
       <item>
        <property name="text">
         <string>最近邻</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>双线性</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>双三次</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lanczos</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
